operators/operator_sdk/java/osdk-java-updating-projects.adoc

```diff
@@ -0,0 +1,25 @@
+:_content-type: ASSEMBLY
+[id="osdk-java-updating-projects"]
+= Updating projects for newer Operator SDK versions
+include::_attributes/common-attributes.adoc[]
+:context: osdk-java-updating-projects
+
+:osdk_ver: v1.25.1
+:osdk_ver_n1: v1.22.0
+
+toc::[]
+
+{product-title} {product-version} supports Operator SDK {osdk_ver}. If you already have the {osdk_ver_n1} CLI installed on your workstation, you can update the CLI to {osdk_ver} by xref:../../../operators/operator_sdk/osdk-installing-cli.adoc#osdk-installing-cli[installing the latest version].
+
+However, to ensure your existing Operator projects maintain compatibility with Operator SDK {osdk_ver}, update steps are required for the associated breaking changes introduced since {osdk_ver_n1}. You must perform the update steps manually in any of your Operator projects that were previously created or maintained with {osdk_ver_n1}.
```

Comment:
I see this in other places, but will note here that the phrase "associated breaking changes" is vague to me. (Ignore my comment if this is a common OS docs phrase.) Does this mean new changes are introduced with this version that break previous projects unless you perform a manual update? This may just be me and my unfamiliarity with this process.

Reply:
To answer your question, updating the Operator SDK is pretty simple. You can just download the latest binary and update any dependencies like Go or Ansible. However, updating an Operator project built with an earlier version of the SDK is more complicated. To make a project compatible with a newer version of the SDK, you have to make manual changes to your configuration files and rebuild the Operator. 

You raise a good point. This introduction could use some clarification. I am going to make a continuos improvement item to clean this up. Since it is repeated across all the Operator assemblies, I want to take it up in another PR. Thank you for the feedback!

Reply:
Thank you for teaching me about this! :)